### DISCUSSION TITLE
Czy ktoś coś ... `elektroda.pl`

### DISCUSSION THREAD

**Discussion by AroKol78** [MajkiIT/polish-ads-filter#22304] [Q&A] [answered]
... wie, ma podobnie lub może pan @gulson-dev mógł by rozwiązać problem
jeszcze kilka dni temu przeglądałem a teraz to
![bugelektroda01](https://github.com/MajkiIT/polish-ads-filter/assets/70756447/4a26b93f-4896-46c9-9da1-6e76d0b69b47)
pobiera `GET` `doc` i na tym koniec.
to jest jedyna strona na której występuje taki problem.

**Comment by gulson-dev** [comment]
Bardzo ciekawe, jakby problem z certyfikatem? Ale nikt nie zgłosił podobnego problemu. Czy byłbyś w stanie podać konfigurację przeglądarki i swoje IP? Sprawdzę logi, mój email do wysłania: elektroda@elektroda.pl
Dziękuję i przepraszam za problemy.

**Reply by AroKol78** [reply]
w podanym zrzucie ekranu na końcu jest podany TLS1.3 w jakim przeglądarka próbuje się połączyć (może nie te szyfrowanie).
te szyfrowanie działa np. github, gitlab i na wielu innych stronach.

**Reply by gulson-dev** [reply]
Dzięki, TLS1.3 wygląd na ostatni, czyli wszystko powinno działać prawidłowo. Jakbyś mi wysłał na email dane tej strony:
https://atomurl.net/myip/
Twoje dane jak IP i przeglądarka będą przetwarzane z najwyższą prywatnością. 
Napiszę do supportu, aby odnaleźli Ciebie w logach, bo nic nie wpada do serwera aplikacyjnego (połączenie jest zrzucane).

**Reply by AroKol78** [reply, marked as answer]
Z mojej perspektywy problem jest pozytywnie rozwiązany - strona działa.
Problem została zidentyfikowany i okazał się dość kuriozalny.

Dzięki za poświęcony czas.

**Reply by gulson-dev** [reply]
Dziękuję za informacje, ja dalej będę męczył wsparcie.

**Reply by AroKol78** [reply]
> dalej będę męczył wsparcie

czytam na forum podanym w mailach, że problem się powtórzył, to znaczy, że wsparcie olało temat.
ostatnio też mi się to powtórzyło, to rozwiązałem podbiciem numeru ff z 120 na 128, ale uważam że nie tędy droga.

**Reply by kgulczuk** [reply]
Zmieniono nam certyfikat na najnowszy, czy jest obecnie jakiś problem? Sprawdziłem nawet na XP pod Firefox 52 i wszystko działa (emulacja przeglądarki Lambdatest). Dzisiaj użytkownik używający MyPal bazujący na Firefox 78 również stwiedził, że jest w porządku.

**Reply by AroKol78** [reply]
> Zmieniono nam certyfikat na najnowszy, czy jest obecnie jakiś problem?

nie ma z tym problemu.
powrócił problem z obrazka z góry przy `120` a przy `128` wszystko u mnie gra
`general.useragent.override.elektroda.pl` `Mozilla/5.0 (%OS_SLICE% rv:128.0) Gecko/20100101 Firefox/128.0 Basilisk/52.9.0`

`%OS_SLICE%` w moim przypadku jest to Windows NT 5.1

**Comment by krystian3w** [comment]
Sprawdź poradniki na aktualizację bazy certyfikatów "TLS" w XP?

**Reply by AroKol78** [reply]
czy sprawdzałem - to nie, wiem, że mam możliwość zaktualizowania bazy certyfikatów, widziałem to np. na msfn, legacyupdate.
mi wystarcza, że przeglądarka obsługuje certyfikaty TLS1.2/1.3 natywnie.

**Comment by krystian3w** [comment]
@gulson-dev AI ma problem z wymową mAh, robi nowy twór `miliapmer `<b>O</b>`dzin`, albo `trzeba tamiętać`.

`https://www.elektroda.pl/rtvforum/topic4128477.html`

w innych miejscach `G`/`P` jest słyszalne.

**Reply by gulson-dev** [reply]
Dzięki, ciekawy przypadek rozwinięcia skrótu mAh. Tak działa od OpenAI, chyba muszę przejść na ElevenLabs.
